SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
```

Comment:
Please remove SECURITY.md or note that we don't care about vulnerabilities in this project and make a statement in README-File too, that this is just experimental